src/main/java/com/faforever/client/game/GameService.java

```diff
@@ -270,14 +265,11 @@ public void changed(ObservableValue<? extends GameStatus> observable, GameStatus
         }
 
         Player currentPlayer = getCurrentPlayer();
-        boolean playerStillInGame = game.getTeams().entrySet().stream()
-            .flatMap(stringListEntry -> stringListEntry.getValue().stream())
-            .anyMatch(playerName -> playerName.equals(currentPlayer.getUsername()));
 
         /*
-          Check if player left the game while it was open, in this case we don't care any longer
+          Check if player still in the game
          */
-        if (newStatus == GameStatus.PLAYING && oldStatus == GameStatus.OPEN && !playerStillInGame) {
+        if (!(currentPlayer.getGame() == game) && game.getStatus() != GameStatus.CLOSED) {
```

Comment:
```suggestion
        if (isCurrentPlayerInGame(game)) {
```

and add the `CLOSED`  check to the method.